spoonbill/writers/xlsx.py

```diff
@@ -57,7 +66,10 @@ def writerow(self, table, row):
         for column, value in row.items():
             col_index = self.col_index[table][column]
             try:
-                sheet.write(self.row_counters[table], col_index, value)
+                if isinstance(value, bool):
```

Comment:
its better to modify value when this condition is satisfied and not touch `sheet.write...`